rotatescreen/display.py

```diff
@@ -1,32 +1,16 @@
+from typing import Dict, List, Tuple
 import win32api
 import win32con
 
 
-def get_displays():
-    displays = [Display(hMonitor) for hMonitor,_,_ in win32api.EnumDisplayMonitors()]
-    return displays
-
-
-def get_primary_display():
-    for display in get_displays():
-        if display.is_primary:
-            return display
-
-
-def get_secondary_displays():
-    displays = [display for display in get_displays() if not display.is_primary]
-    return displays
-
-
 class Display:
-
     def __init__(self, hMonitor):
         self.hMonitor = hMonitor
 
     def __repr__(self):
         return f"<'{self.device_description[0]}' object>"
 
-    def rotate_to(self, degrees):
+    def rotate_to(self, degrees) -> None:
```

Comment:
Perhaps worth adding parameter hints also?

```suggestion
    def rotate_to(self, degrees: int) -> None:
```